core/integration/services_test.go

```diff
@@ -953,7 +953,7 @@ func (ServiceSuite) TestDirectoryEntries(ctx context.Context, t *testctx.T) {
 
 	entries, err := c.Git(repoURL, dagger.GitOpts{ExperimentalServiceHost: gitDaemon}).
 		Branch("main").
-		Tree().
+		Tree(dagger.GitRefTreeOpts{DiscardGitDir: true}).
```

Comment:
nit: I would be cool with updating the assertions instead to just include `.git`, since setting this here makes it feel more significant than it really is (I think?)